apps/api/src/app/bridge/usecases/sync/sync.usecase.ts

```diff
@@ -142,46 +158,79 @@ export class Sync {
     });
   }
 
-  private async createWorkflows(
+  private async processWorkflows(
     command: SyncCommand,
     workflowsFromBridge: DiscoverWorkflowOutput[]
   ): Promise<NotificationTemplateEntity[]> {
     return Promise.all(
       workflowsFromBridge.map(async (workflow) => {
-        const workflowExist = await this.notificationTemplateRepository.findByTriggerIdentifier(
-          command.environmentId,
-          workflow.workflowId
+        let savedWorkflow = await this.upsertWorkflow(command, workflow);
+
+        const validatedWorkflowWithIssues = await this.workflowUpdatePostProcess.execute({
+          user: {
+            _id: command.userId,
+            environmentId: command.environmentId,
+            organizationId: command.organizationId,
+          } as UserSessionData,
+          workflow: {
+            ...savedWorkflow,
+            userPreferences: null,
+            defaultPreferences: this.getWorkflowPreferences(workflow),
+          },
+        });
+
+        savedWorkflow = await this.updateWorkflowUsecase.execute(
+          UpdateWorkflowCommand.create({
+            ...validatedWorkflowWithIssues,
+            id: validatedWorkflowWithIssues._id,
+            type: WorkflowTypeEnum.BRIDGE,
+            environmentId: command.environmentId,
+            organizationId: command.organizationId,
+            userId: command.userId,
+          })
         );
```

Comment:
this part is responsible for updating the issues on the workflow


![image](https://github.com/user-attachments/assets/9b12eece-580a-429b-9ccf-bae5629f3eab)